.github/workflows/1-preparing.yml

```
@@ -1,6 +1,4 @@
-name: Step 1
-
-
+name: Step 1 # Preparing to make your extension
```

This comment looks copy pasted from some exercise? It's not generic for the template

```suggestion
name: Step 1 
```